src/components/BnTextarea/BnTextarea.vue

```diff
@@ -47,6 +48,7 @@ const attrs = useAttrs();
     :class="`bn-textarea--${props.color} ${attrs.class ? attrs.class : ''}`"
   >
     <textarea
+      v-bind="attrsWithoutClass"
```

Comment:
because the attributes are applied by default on the root, the parent div in this case. So things like disabled="true" were not affecting the actual textarea.
This wasn't applied before because there was no parent div at some point, and in a PR I had to add it but I forgot this

Reply:
I understand. I think you have to [disable inherited](https://vuejs.org/guide/components/attrs.html#disabling-attribute-inheritance) attrs to not apply it to the parent too.

Reply:
I understand. I think you have to [disable inherited](https://vuejs.org/guide/components/attrs.html#disabling-attribute-inheritance) attrs to not apply it to the parent too.

Reply:
uh you´re right! And I didn't know since 3.3 you could disable it in script setup 😮 nice

Reply:
done!